### DISCUSSION TITLE
wierd font in nextjs.org

### DISCUSSION THREAD

**Discussion by kiyan-alav** [vercel/next.js#52731] [Help]
### Summary

Hello 
It's about 2 weeks, I switched from CRA to Vite. I have strange bug on all my browsers and this bug is in "nextjs.org" too.
I searched all the Internet to find problem but nothing there!!!
![bug](https://github.com/vercel/next.js/assets/81843676/a9d2f5b9-9054-463f-9f78-b4dc5b24e14c)
This photo is for about 2 weeks ago
This bug is killing me!! If someone had same situation please give me a solution

**Comment by Bhuniasourav** [comment]
![image](https://github.com/vercel/next.js/assets/106818556/74c050f7-fca0-4eab-a42c-ee868903fe3a)
i have same problem but i am also not get solution.
